src/index.ts

```diff
@@ -9,10 +9,12 @@ type PersistOptions = {
   selector?: string
   keyPrefix?: string
   storage?: Pick<Storage, 'getItem' | 'setItem'>
+  scope?: ParentNode
```

Comment:
considering this with https://github.com/github/session-resume/pull/20#discussion_r1464013611, `scope` probably would not make sense combined with `fields` so you'd want to account for that as well

Reply:
Thank you for that guidance! In that case, I think I'm going to close this PR in favor of #20, then account for both options in one atomic commit.